src/screens/toDoList/ToDoList.tsx

```diff
@@ -1,16 +1,17 @@
 import React from 'react';
-import {FlatList, ListRenderItem, Modal, Text, View} from 'react-native';
+import {Modal, Text, View} from 'react-native';
 import ToDoListRow from './components/ToDoListRow';
-import FloatingButton from '../../ui/FloatingButton';
+import FloatingButton from '../../ui/floatingButton/FloatingButton';
```

Comment:
```suggestion
import {FloatingButton} from '../../ui/floatingButton';
```

export FloatingButton from index.ts